filedescriptor/src/windows.rs

```diff
@@ -255,6 +261,36 @@ impl FileDescriptor {
         }
         Ok(())
     }
+
+    pub(crate) fn redirect_stdio_impl<F: AsRawFileDescriptor>(
+        f: &F,
+        stdio: StdioDescriptor,
+    ) -> anyhow::Result<Self> {
+        let std_handle = match stdio {
+            StdioDescriptor::Stdin => STD_INPUT_HANDLE,
+            StdioDescriptor::Stdout => STD_OUTPUT_HANDLE,
+            StdioDescriptor::Stderr => STD_ERROR_HANDLE,
+        };
+
+        let raw_std_handle = unsafe { GetStdHandle(std_handle) } as *mut _;
+        let std_original = unsafe { FileDescriptor::from_raw_handle(raw_std_handle) };
+
+        let cloned_handle = OwnedHandle::dup(f)?;
+        if unsafe {
+            SetStdHandle(
+                std_handle,
+                cloned_handle.into_raw_handle() as *mut winapi::ctypes::c_void,
```

Comment:
I think you can simplify this:

```suggestion
                cloned_handle.into_raw_handle() as *mut _,
```

Reply:
Done